README.md

```diff
@@ -5,4 +5,70 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/ontariosystems/isclib)](https://goreportcard.com/report/github.com/ontariosystems/isclib)
 [![GoDoc](https://godoc.org/github.com/ontariosystems/isclib?status.svg)](https://godoc.org/github.com/ontariosystems/isclib)
 
-Go library for interacting with InterSystems Corporation products like Caché, Ensemble, and IRIS Data Platform
\ No newline at end of file
+Go library for interacting with InterSystems Corporation products like Caché, Ensemble, and IRIS Data Platform
+
+It provides methods for determining if ISC products are installed and for interacting with instances of them
+
+### Checking for available ISC commands
+
+```go
```

Comment:
I would consider adding these to the go documentation so they show up in the [generated docs](https://godoc.org/github.com/ontariosystems/isclib) and then link to the go docs from the `README.md`.

Here's an example of some docs like this.
https://godoc.org/github.com/golang/protobuf/proto
https://github.com/golang/protobuf/blob/master/proto/lib.go

Reply:
Moved, hopefully correctly

Reply:
Looks right to me. 😬